polaris/polaris/management/commands/process_pending_deposits.py

```diff
@@ -35,151 +34,130 @@
     registered_fee_func,
     calculate_fee,
 )
-from polaris.models import Transaction
+
+from polaris.exceptions import (
+    TransactionSubmissionPending,
+    TransactionSubmissionBlocked,
+    TransactionSubmissionFailed,
+)
+
+from polaris.models import Transaction, PolarisHeartbeat
 from polaris.utils import getLogger
 
+
 logger = getLogger(__name__)
 
-TERMINATE = False
-"""
-SIGINT and SIGTERM signals to this process set TERMINATE to True,
-and once all pending tasks complete, the process exits.
-Only relevant if the --loop option is specified.
-"""
+SUBMIT_TRX_QUEUE = "SUBMIT_TRX_QUEUE"
+CHECK_ACC_QUEUE = "CHECK_ACC_QUEUE"
 
+DEFAULT_HEARTBEAT = 5
 DEFAULT_INTERVAL = 10
-"""
-The default amount of time to sleep before querying for transactions again
-Only used if the --loop option is specified.
-"""
 
+PROCESS_PENDING_DEPOSITS_LOCK_KEY = "PROCESS_PENDING_DEPOSITS_LOCK"
 
-class Command(BaseCommand):
-    """
-    The process_pending_deposits command handler.
-    """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        signal.signal(signal.SIGINT, self.exit_gracefully)
-        signal.signal(signal.SIGTERM, self.exit_gracefully)
-
-    @staticmethod
-    def exit_gracefully(*_):  # pragma: no cover
-        logger.info("Exiting process_pending_deposits...")
-        module = sys.modules[__name__]
-        module.TERMINATE = True
+class PolarisQueueAdapter:
+    def __init__(self, queues):
+        self.queues: dict[str, asyncio.Queue] = {}
+        for queue in queues:
+            self.queues[queue] = asyncio.Queue()
 
-    @staticmethod
-    def sleep(seconds):  # pragma: no cover
-        for _ in range(seconds):
-            if TERMINATE:
-                break
-            time.sleep(1)
+    def populate_queues(self):
+        """
+        populate_queues gets called to read from the database and populate the in-memory queues
+        """
+        logger.info("initializing queues from database...")
```

Comment:
Can we update all logging statements to `logger.debug()` and only use `.info()` for events specific to deposit transactions, such as errors, account creation, etc.?